From b50c6442e43d79471a31a2a202d3e50c0557446f Mon Sep 17 00:00:00 2001
From: Sun Haiyong <youbest@sina.com>
Date: Sat, 4 Sep 2021 15:06:43 +0200
Subject: sigsegv: Improve cross-compilation support for LoongArch CPU.

* m4/stack-direction.m4 (SV_STACK_DIRECTION): Assume the stack grows
down on LoongArch.
---
 m4/stack-direction.m4 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 9328725..e682be9 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -1,4 +1,4 @@
-# stack-direction.m4 serial 7
+# stack-direction.m4 serial 8
 dnl Copyright (C) 2002-2021 Free Software Foundation, Inc.
 dnl This file is free software, distributed under the terms of the GNU
 dnl General Public License.  As a special exception to the GNU General
@@ -32,6 +32,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       i?86 | x86_64 | \
       i860 | \
       ia64 | \
+      loongarch* | \
       m32r | \
       m68* | \
       m88k | \
-- 
cgit v1.1
